lib/rules/require-capitalized-comments.js

```diff
@@ -66,25 +137,44 @@ module.exports.prototype = {
     check: function(file, errors) {
         var inTextBlock = null;
 
+        var exceptions = this._exceptions || {};
+        var hasExceptions = this._hasExceptions;
+
         var letterPattern = require('../../patterns/L');
         var upperCasePattern = require('../../patterns/Lu');
 
         file.getComments().forEach(function(comment) {
-            var stripped = comment.value.replace(/[\n\s\*]/g, '');
```

Comment:
please revert the moving of these declarations. There is no reason I can see to not initialize them here.